README.md

```diff
@@ -74,6 +74,10 @@ The Astrobee platform is licensed under the Apache License, Version 2.0 (the
 "License"); you may not use this file except in compliance with the License. You
 may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
 
+### Sema Comment
+
+This code is wonderful.
+
```

Comment:
This code is awesome.

---
**Sema Reaction:** :trophy: Awesome

Reply:
This variable needs a fix.

---
**Sema Reaction:** :hammer_and_wrench: Fix

Reply:
very clear code

---
**Sema Reaction:** :ok_hand: Looks good | **Sema Tags:** Readable